test/mappings/ConstraintsSpec.scala

```diff
@@ -235,7 +235,6 @@ class ConstraintsSpec extends SpecBase with Constraints {
 
 trait SetUp {
   val spaces: Some[String]                            = Some("   ")
```

Comment:
generic val like spaces etc can probably be moved to the TestData or the Utils file